lib/cn/omf.h

```diff
@@ -31,7 +32,7 @@
  */
 #define WBT_NODE_SIZE PAGE_SIZE /* must equal system page size */
 
-#define WBT_TREE_MAGIC ((u32)0x4a3a2a1a)
+#define WBT_TREE_MAGIC UINT32_C(0x4a3a2a1a)
```

Comment:
Technically, this could generate a 64-bit constant, seems like you want to ensure it's always 32-bits regardless of compilation environment, right?  Unless you have defined your type to be uint_least32_t as well, which you shouldn't because we want fixed-width types in omf...

Reply:
Yea...reading up on this is rather interesting. I will revert these particular changes.